.github/CODEOWNERS

```
@@ -9,3 +9,14 @@ app/components/UI/Ramp/              @MetaMask/ramp @MetaMask/mobile-platform
 app/reducers/fiatOrders/             @MetaMask/ramp @MetaMask/mobile-platform
 app/components/Views/confirmations   @MetaMask/confirmations-ux @MetaMask/mobile-platform
 ppom                                 @MetaMask/confirmations-ux @MetaMask/mobile-platform
+app/actions/sdk   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/components/Approvals/WalletConnectApproval   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/components/Views/SDK   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/components/Views/WalletConnectSessions   @MetaMask/sdk-devs @MetaMask/mobile-platform
+app/core/BackgroundBridge/WalletConnectPort.ts   @MetaMask/sd
```

@omridan159 You have a typo here
```suggestion
app/core/BackgroundBridge/WalletConnectPort.ts   @MetaMask/sdk-devs @MetaMask/mobile-platform
```